openstack/networking/v2/extensions/lbaas_v2/loadbalancers/results.go

```diff
@@ -0,0 +1,128 @@
+package loadbalancers
+
+import (
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// LoadBalancer is the primary load balancing configuration object that specifies
+// the virtual IP address on which client traffic is received, as well
+// as other details such as the load balancing method to be use, protocol, etc.
+// This entity is sometimes known in LB products under the name of a "virtual
+// server", a "vserver" or a "listener".
```

Comment:
can we maybe delete the last sentence? The only reason is that this new extension has very precise terminology and referencing listeners in different contexts has the potential to confuse